kong.conf.default

```diff
@@ -1548,7 +1548,7 @@
 
 #dns_error_ttl = 1               # TTL in seconds for error responses.
 
-#dns_no_sync = on                # If enabled, then upon a cache-miss every
+#dns_no_sync = off               # If enabled, then upon a cache-miss every
```

Comment:
In PR #11869 we have turn on this flag, should we add another change log for it?

Reply:
I think we need to add this changed info to `[changelog/unreleased/kong/fix_dns_blocking.yml](https://github.com/Kong/kong/pull/11900/files#diff-dcaf6440867320b2760937851ee0fc8d15a37236ead390e382c9c956af88ee7d)`

```
message: Eliminate asynchronous timer in syncQuery() to prevent hang risk
```

->

```
message: Eliminate asynchronous timer in syncQuery() to prevent hang risk
                 The default value of `dns_no_sync` option has been changed to `off`
```

Reply:
hi @chronolaw See my new fix: https://github.com/Kong/kong/pull/11900/commits/d3ee61ce617694ee11da84742da48820f6053622. If this fix is ok, pls help to resolve this comment, ty!

Reply:
Please create two Changelog entries for these two changes.

Reply:
KK

Reply:
fixed with two changelog introduced

Reply:
great.